cmd/kes/update.go

```diff
@@ -0,0 +1,120 @@
+// Copyright 2021 - MinIO, Inc. All rights reserved.
+// Use of this source code is governed by the AGPLv3
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"crypto/tls"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net"
+	"net/http"
+	"os"
+	"runtime"
+	"time"
+
+	"github.com/cheggaaa/pb/v3"
+	"github.com/minio/selfupdate"
+)
+
+func getUpdateTransport(timeout time.Duration) http.RoundTripper {
```

Comment:
Why are we using the same timeout for everything? That does not seem reasonable

Reply:
There is no reasonable reason to have different timeouts for each since its short-lived transport. All we need is a top level timeout.